src/EXTRA-COMPUTE/compute_stress_cartesian.cpp

```diff
@@ -229,35 +230,16 @@ void ComputeStressCartesian::init_list(int /* id */, NeighList *ptr)
 
 void ComputeStressCartesian::compute_array()
 {
-  int i, j, ii, jj, inum, jnum, itype, jtype;
-  int bin, bin1, bin2;
-  tagint itag, jtag;
-  double xtmp, ytmp, ztmp, delx, dely, delz;
-  double rsq, fpair, factor_coul, factor_lj;
-  int *ilist, *jlist, *numneigh, **firstneigh;
-
   double **x = atom->x;
   double **v = atom->v;
-  double *mass = atom->mass;
```

Comment:
Please keep these cached values from Atom class members and other pointers and values. It makes for an easier read of the code, keeps it consistent with other parts of LAMMPS, and has a (small) performance benefit since it reduces the need to chase pointers in every loop iteration.

Reply:
I sincerely doubt it makes any difference for performance (with modern compilers, but I'll be happy to be proved wrong ;)), but I'll revert the copied pointers for consistency with the rest of LAMMPS if so desired. :)